es_ES/layout.md

```diff
@@ -0,0 +1,68 @@
+
+
+* Placeholder for TOC
+{:toc}
+
+## Introducción
+
+En Flutter, los widgets son renderizados por sus objetos [`RenderBox`](https://docs.flutter.io/flutter/rendering/RenderBox-class.html) subyacentes. Los cuadros generadores obtienen restricciones por parte de sus padres y se clasifican dentro de esas restricciones. 
+Las restricciones consisten en valores mínimos y maximos para el ancho y alto; los tamaños consisten en un ancho y alto específicos.
+
+En general, hay tres tipos de cajas, en términos de cómo manejar
+sus restricciones:
+
+- Aquellos que tratan de ser los más grandes posible.
+  Por ejemplo, las cajas usadas por [`Center`](https://docs.flutter.io/flutter/widgets/Center-class.html) y [`ListView`](https://docs.flutter.io/flutter/widgets/ListView-class.html).
+- Aquellos que tratan de tener el mismo tamaño que su hijo.
+  Por ejemplo, las cajas usadas por [`Transform`](https://docs.flutter.io/flutter/widgets/Transform-class.html) y [`Opacity`](https://docs.flutter.io/flutter/widgets/Opacity-class.html).
+- Aquellos que tratan de tener un tamaño en particular.
+  Por ejemplo, las cajas usadas por [`Image`](https://docs.flutter.io/flutter/dart-ui/Image-class.html) y [`Text`](https://docs.flutter.io/flutter/widgets/Text-class.html).
+
+Algunos widgets, por ejemplo [`Container`](https://docs.flutter.io/flutter/widgets/Container-class.html), varian de tipo en tipo basados en sus argumentos de construcción. En el caso de [`Container`](https://docs.flutter.io/flutter/widgets/Container-class.html), este por defecto trata de ser tan grande como sea posible, pero si tu colocas un valor en `width`, para esa instancia, trata de honrar ese valor y tomar un tamaño particular.
+
+Otras, por ejemplo [`Row`](https://docs.flutter.io/flutter/widgets/Row-class.html) y [`Column`](https://docs.flutter.io/flutter/widgets/Column-class.html) (cajas flexibles) varían basándose en las restricciones que se les otorgue, como se describe a continuación en la sección "Flex".
+
+Las restricciones a veces son "estrictas", lo que significa que no dejan espacio para que la caja en el renderizado decida un tamaño
+(e.g. si el mínimo y máximo  de ancho son el mismo, se dice que tienen un ancho estricto). El widget de ejemplo principal de esta `App`, es restringido por la clase
+[`RenderView`](https://docs.flutter.io/flutter/rendering/RenderView-class.html): la caja usada por el hijo devuelto por la función principal de la aplicación [`build`](https://docs.flutter.io/flutter/widgets/State/build.html) le da una restricción que lo fuerza a llenar exactamente el área del contenido de la aplicación  (comúnmente toda la aplicación). Muchas de las cajas en Flutter, especialmente aquellas que tienen un único hijo, van a pasar sus restricciones a sus hijos. Eso significa que si anida un grupo de cajas dentro de otra en la raíz del árbol de renderizado de tu aplicación, todas encajarán exactamente entre sí, forzadas por las restricciones estrictas.
```

Comment:
the box used by the child returned by the
application's [`build`](https://docs.flutter.io/flutter/widgets/State/build.html) function is given a constraint that forces it to exactly fill the application's content area (typically, the entire
screen)

la caja usada por el hijo retornado por la función [`build`](https://docs.flutter.io/flutter/widgets/State/build.html) de la aplicación, le da una restricción que lo fuerza a rellenar exactamente el área de contención de la aplicación ( normalmente, toda la pantalla)